### ISSUE TITLE
Preprocessing - input_range and mean/std

### ISSUE THREAD

**Issue by karandwivedi42** [Cadene/pretrained-models.pytorch#66]
For example in se-resnet50, the input_range in [0,1]. So the input to model is `input - mean / std` and not actually [0,1], right?

```python
def train_transform(x):
    x = TF.resize(x, 256)
    i, j, h, w = transforms.RandomCrop.get_params(x, [224, 224])
    x = TF.crop(x, i, j, h, w)
    if TF.random.random() > 0.5:
        x = TF.hflip(x)
    x = TF.to_tensor(x)
    x = TF.normalize(x, [0.485, 0.456, 0.406], [0.229, 0.224, 0.225])
    return x

def val_transform(x):
    x = TF.resize(x, 224)
    x = TF.center_crop(x, 224)
    x = TF.to_tensor(x)
    x = TF.normalize(x, [0.485, 0.456, 0.406], [0.229, 0.224, 0.225])
    return x
```

**Comment by Cadene** [comment]
@karandwivedi42 Yes, [0,1] means that the original pixels must be scaled between 0 and 1. With the Caffe models it is usually [0,255] and we use a hack: https://github.com/Cadene/pretrained-models.pytorch/blob/master/pretrainedmodels/utils.py#L29

**Comment by karandwivedi42** [comment]
Thanks! Just to be extra sure, the preprocessing code in the original post is correct, right?

**Comment by Cadene** [comment]
It seems right to me :)

**Comment by pyaf** [comment]
Hey, @Cadene Could you please explain to me what's the significance of [this](https://github.com/Cadene/pretrained-models.pytorch/blob/master/pretrainedmodels/models/nasnet_mobile.py#L32) line? I'm normalizing my images with the mean and std of imagenet dataset. What is this [0.5, 0.5, 0.5] meant to be?

**Comment by Cadene** [comment]
The mean used by google team to normalize the images.
They didn't use the "correct" mean and std

**Comment by pyaf** [comment]
Thanks for quick response. So, should I use [0.5, 0.5, 0.5] as my mean and std instead?

**Comment by Cadene** [comment]
Yes :)
